modules/cluster-openshift-controller-manager-operators.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+[id="cluster-openshift-controller-manager-operator_{context}"]
+= Cluster OpenShift Controller Manager Operator
+
+[discrete]
+== Purpose
+
+The Cluster OpenShift Controller Manager Operator installs and maintains the `OpenShiftControllerManager` Custom Resource in a cluster and can be viewed with:
+
+----
+$ oc get clusteroperator openshift-controller-manager -o yaml
+----
+
+The Custom Resource Definition `openshiftcontrollermanagers.operator.openshift.io`
+can be viewed in a cluster with:
+
+----
+$ oc get crd openshiftcontrollermanagers.operator.openshift.io -o yaml
+----
+
+[discrete]
+== Project
+
+link:https://github.com/openshift/cluster-openshift-controller-manager-operator[cluster-openshift-controller-manager-operator]
```

Comment:
Acked by QE.

Reply:
Thank you! The content here will be built out soon, likely in a separate PR